scripts/docs/loadExample.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-dynamic-require, global-require, no-console */
+import path from "path";
+import chalk from "chalk";
+
+export default function loadExample(file) {
+  let Component;
+  try {
+    const example = path.resolve(file, "../example.js");
+    Component = require(example);
+  } catch (e) {
+    // ignore eror requiring missing file
+  }
+
+  if (!Component) {
+    Component = require(file);
+  }
+
+  if (typeof Component === "object") {
+    Component = Component.default;
+  }
+
+  if (!Component) {
+    console.log(chalk.red("Error loading component:"), file);
+  }
+
```

Comment:
is it necessary to have two `if (!Component)`?

Reply:
second is "if there still isn't a component after loading the default", so yes

Reply:
1. try load example.js
2. if this didn't load, index.js
3. if this isn't a component, look for default export
4. if still no component then log

Reply:
ah true cool